README.md

```diff
@@ -59,15 +57,17 @@ Unpack the medley loadups file
 
 ### Setting up X
 
-Medley Interlisp currently needs an X-Server to manage its display. Most Linux desktops have one. Windows 11 with WSL includes an X-Server. For Windows 10 with WSL2, there are a number of open-source X servers; for example vcxsrv.
+In many configurations, Medley uses an X-Server to manage its display. Most Linux desktops have one. Windows 11 with WSL includes an X-Server. For Windows 10 with WSL2, there are a number of open-source X servers; for example vcxsrv.
 
 Mac users should get [XQuartz from XQuartz.org](https://xquartz.org/releases).
 
 Medley manages the display entirely, doesn't use X fonts and manages it's own window system.
 
-If you have a high-resolution display, note that much of the graphics was designed for a low-resolution display, so an X-server that does "pixel doublilng" is best. (E.g., Raspberry Pi does pixel doubling on 4K displays.) It also presumes you have a 3-button mouse; the scroll-wheel on some mice act as one with some difficulty.) XQuartz Preferences/Input has "Emulate three button mouse" option.
+If you have a high-resolution display, note that much of the graphics was designed for a low-resolution display, so an X-server that does "pixel doublilng" is best. (E.g., Raspberry Pi does pixel doubling on 4K displays.)
+
+Medley presumes you have a 3-button mouse; the scroll-wheel on some mice act as one with some difficulty.) Go  into XQuartz Preferences/Input and check "Emulate three button mouse" option.
```

Comment:
Line 68: Unmatched right parenthesis: **difficulty.)**
Line 115: **at least in the library, lispusers.** (poor grammar) => **at least in the library and lispusers directories.**

Reply:
Apparently, I cannot add the comment to lines that were not modified. So, the line 115 change is here.

Reply:
Also, "the scroll-wheel on some mice act**s** as one, with some difficulty.

I'll fix your comments and push a new version (BTW, Matt - I believe that you can update these branches too)

Reply:
I think these are resolved now.  Marking it as such.